_maps/RandomRuins/LavaRuins/lavaland_surface_colossus_arena.dmm

```diff
@@ -0,0 +1,44 @@
+"a" = (/turf/closed/mineral/volcanic/lava_land_surface,/area/ruin/powered{name = "Ominous Lair"})
```

Comment:
I still dislike this sort thing but maybe I am crazy letting them roam about like they are